sympy/logic/boolalg.py

```diff
@@ -2373,7 +2401,7 @@ def _input_to_binlist(inputlist, variables):
             nonspecvars = list(variables)
             for key in val.keys():
                 nonspecvars.remove(key)
-            for t in product([0, 1], repeat=len(nonspecvars)):
+            for t in product((0, 1), repeat=len(nonspecvars)):
```

Comment:
This is fine but in general it would be better to make these kinds of changes in a separate PR from one that actually changes behaviour or fixes bugs etc.

Reply:
You are correct. Tends to just see lots of things and, a bit manically, change them...